lib/src/code_field/code_controller.dart

```diff
@@ -803,7 +853,17 @@ class CodeController extends TextEditingController {
     bool? withComposing,
   }) {
     // TODO(alexeyinkin): Return cached if the value did not change, https://github.com/akvelon/flutter-code-editor/issues/127
-    return lastTextSpan = _createTextSpan(context: context, style: style);
+    lastTextSpan = SearchResultHighlightedBuilder(
+      searchResult: searchResult,
+      rootStyle: style,
+    ).build(
+      _createTextSpan(
+        context: context,
+        style: style,
+      ),
+    );
```

Comment:
```suggestion
    final spanBeforeSearch = _createTextSpan(
      context: context,
      style: style,
    );

    lastTextSpan = SearchResultHighlightedBuilder(
      searchResult: searchResult,
      rootStyle: style,
    ).build(spanBeforeSearch);
```

Reply:
Why do we pass `searchResult` and `style` in constructor but pass `spanBeforeSearch` in `build()`?